ols/utils/logger.py

```diff
@@ -41,8 +41,8 @@ class Logger:
     def __init__(
         self,
         logger_name: str = "default",
-        log_level: int = logging.INFO,
-        logfile: str = None,
+        log_level: int | str = logging.INFO,
```

Comment:
this should not be either/or.  

if we are representing/passing the loglevel as a string in some places and an int in others, let's straighten that out now.

I'd expect that users will provide the loglevel as a string (e.g. `info`) and then we will validate that they've provided as a valid loglevel that the loghandler understands (possibly by converting it to an int and then using that int value consistently internally)

Reply:
well the standard logging module supports both. We can indeed remove integer one as it is not user-friendly

Reply:
yeah let's just support string from a user-facing/config perspective.

Reply:
but ok, can change it

Reply:
updated @bparees